src/libs/ords-pcss-client/ords.pcss.swagger.yaml

```diff
@@ -96,7 +96,28 @@ paths:
           in: query
           type: string
           description: Implicit parameter  
-          pattern: '^[^/]+$'      
+          pattern: '^[^/]+$'
+  '/civil/file-content/document-support/':
+    get:
+      tags:
+      - PCSS-CIVIL
+      security:
+      - basicAuth: []      
+      description: Get the civil file content document support data
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            type: object
```

Comment:
ref must be at this level

Reply:
Removed the object and prop fields and put the ref under schema:  I think that's what you meant?